clients/src/main/java/org/apache/kafka/common/requests/JoinGroupResponse.java

```diff
@@ -65,7 +66,17 @@ public JoinGroupResponse(short errorCode,
                              String memberId,
                              String leaderId,
                              Map<String, ByteBuffer> groupMembers) {
-        super(new Struct(CURRENT_SCHEMA));
+        this(CURRENT_VERSION, errorCode, generationId, groupProtocol, memberId, leaderId, groupMembers);
```

Comment:
Just a quick note to show the risk of this pattern, I did have an incorrect usage of the old constructor in `KafkaApis`.